src/main/scala/com/stratio/hermes/helpers/HermesConsoleHelper.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2016 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.hermes.helpers
+
+import com.stratio.hermes.actors.HermesClientActor
+import jline.console.ConsoleReader
+
+case class HermesConsoleHelper(client: HermesClientActor) {
+
+  lazy val reader = createDefaultReader()
+
+  //scalastyle:off
+  def parseLines(hermesConfig: Option[String] = None,
+                 kafkaConfig: Option[String] = None,
+                 template: Option[String] = None): Unit = {
+    reader.readLine.trim match {
+      case "set hermes" =>
+        println("Press Control + D to finish")
+        val parsedBlock = Option(parseBlock())
+        reader.setPrompt("hermes> ")
+        parseLines(parsedBlock, kafkaConfig, template)
+
+      case "set kafka" =>
+        println("Press Control + D to finish")
+        val parsedBlock = Option(parseBlock())
+        reader.setPrompt("hermes> ")
+        parseLines(hermesConfig, parsedBlock, template)
+
+      case "set template" =>
+        println("Press Control + D to finish")
+        val parsedBlock = Option(parseBlock())
+        reader.setPrompt("hermes> ")
+        parseLines(hermesConfig, kafkaConfig, parsedBlock)
+
+      case value if value.startsWith("start") =>
```

Comment:
This case and the next one are also very similar, could you please refactor and extract the common code to a method?